pkg/arvo/sys/zuse.hoon

```diff
@@ -6469,8 +6469,16 @@
       ;~  plug
         ;~(pfix (plus whit) name)
         ;~  pose
-          (ifix [;~(plug tis yel) yel] (star ;~(less yel escp)))
-          (ifix [;~(plug tis say) say] (star ;~(less say escp)))
+          %+  ifix
+          :_  yel
+          ;~(plug ;~(pose ;~(plug whit tis) tis) ;~(pose ;~(plug whit yel) yel))
+          (star ;~(less yel escp))
+          ::
+          %+  ifix
+          :_  say
+          ;~(plug ;~(pose ;~(plug whit tis) tis) ;~(pose ;~(plug whit say) say))
+          (star ;~(less say escp))
+          ::
```

Comment:
Also formatting here should be a bit tidier. Might also consider throwing this in a `|^` and factoring out all of the parsing logic that's identical between `yel` and `say` cases, but not sure if that gets you enough gains to be worth it.

```suggestion
       ::
          %+  ifix
            :_  say
            ;~(plug (ifix [. .]:(star whit) tis) say)
          (star ;~(less say escp))
        ::
```

Reply:
Fixed the formatting and tested your stuff, it works and is better!